helm/kyverno-policies-ux/templates/org-giantswarm-namespace-cluster-block.yaml

```diff
@@ -0,0 +1,23 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: block-all-clusters-org-giantswarm
```

Comment:
That this policy is being added is a bit weird. 

I'm ok with it but maybe you want to check with the owner if this is intended.